src/ducc0/fft/fft.h

```diff
@@ -560,7 +630,7 @@ template<size_t N> class multi_iter
 
   public:
     multi_iter(const fmav_info &iarr, const fmav_info &oarr, size_t idim,
-      size_t nshares, size_t myshare)
+      size_t lo, size_t hi)
```

Comment:
This is one of the main changes: from `myshare/nshares` to `[lo, hi]`.

Reply:
That's fine. Related to the discussion above, this pushes us towards using larger chunk sizes, since constructing a `multi_iter` is not very cheap, and we construct one `multi_iter` per chunk. (At the moment this is perfectly fine, but something to keep in mind when switching to dynamic scheduling).

Reply:
Yes, we wouldn't want to be calling this separately for every item `[lo, lo + 1]`.  It's similar to if you had specified a very large number of threads in `DUCC0_NUM_THREADS`.

For context, Eigen's (and TF's) threadpool does recursive splitting.  We start with the range `[0, nwork]`, then use a set of heuristics and a cost function to decide when to stop subdividing and execute the chunk.  This is not necessarily based on the total number of threads, and we don't know a priori how many total chunks there will be (the scheduling heuristic is a function of total work, cost of that work in cycles, and number of threads available at the time).